### ISSUE TITLE
Ogre Magi's Ignite doesn't throw 2 ignites if all nearby targets are already ignited.

### ISSUE THREAD

**Issue by PhillipSF** [ValveSoftware/Dota2-Gameplay#8615]
### Ability name

Ignite

### Description

Casting ignite on an enemy that's already ignited with another nearby enemy that's also ignited will only throw 1 ignite on the intial target instead of throwing 2 ignites, 1 on the intial target, and 1 on the other target.
And if it multicasts, it will also just throw 1 ignite but at a different target from the last.

This bug also happens with multicasted ignites on targets that have just been ignited by the intial ignite.

How to replicate bug:
1. Have 2 or more enemy targets
2. Ignite them all
3. Try to ignite them more
4. You'll start throwing single ignites

### Example Match ID (and possibly Timestamp)

N/A

**Comment by commentator24** [comment]
if not mistaken the duration is longer on same target

**Comment by PhillipSF** [comment]
> 

No, that's normal behaviour, the target gets the standard +8 from getting hit from an ignite. They don't get hit twice from a single ignite for +16.

**Comment by Daar543** [comment]
Last time I checked, the duration gets increased. But even if not, you should not be forbidden from using the spell twice.

**Comment by PhillipSF** [comment]
That's not how the issue goes.
It doesn't prevent you from casting ignite on the target, it just fails to throw the secondary ignite when all nearby targets are ignited, so you only see 1 ignite ball thrown each volley.
Also, if you get a multicast on ignite, that multicast will always pick a different target from the cast before, so it's impossible to get a full x4 multicast ignite on one target while there are other enemies nearby.

**Comment by TheChetu** [comment]
Still bugged as of 2023-08-09 sadly.
